lib/translations.json

```diff
@@ -3525,6 +3489,226 @@
     {
       "key": "pages/meta/behind/gallery",
       "value": "Hinter den Kulissen"
+    },
+    {
+      "key": "package/PROLONG/title",
+      "value": "Verlängern"
+    },
+    {
+      "key": "package/PROLONG/description",
+      "value": "Eine funktionierende Demokratie braucht funktionierende Medien. Und dafür braucht es nicht nur Journalistinnen und Journalisten, sondern auch Sie. Als Leser. Als Bürgerin. Als Mensch, der bereit ist, unabhängigen Journalismus zu finanzieren."
+    },
+    {
+      "key": "package/PROLONG/price",
+      "value": "{formattedCHF} pro Jahr"
+    },
+    {
+      "key": "package/PROLONG/price/give",
+      "value": "{formattedCHF}"
+    },
+    {
+      "key": "option/PROLONG/ABO/label",
+      "value": "Reguläre Verlängerung"
+    },
+    {
+      "key": "option/PROLONG/ABO/label/give",
+      "value": "Verlängerung schenken"
+    },
+    {
+      "key": "option/PROLONG/BENEFACTOR_ABO/label",
+      "value": "Gönnerin oder Gönner"
+    },
+    {
+      "key": "option/PROLONG/resetGroup",
+      "value": "Jetzt nicht"
+    },
+    {
+      "key": "memberships/manage/reactivate",
+      "value": "Kündigung zurücknehmen"
+    },
+    {
+      "key": "memberships/manage/prolong/link",
+      "value": "Mitgliedschaft und Abonnement verlängern"
+    },
+    {
+      "key": "memberships/manage/prolong/awaiting",
+      "value": "Sie haben bereits eine weitere Mitgliedschaft, diese wird automatisch zur Verlängerung genutzt."
+    },
+    {
+      "key": "memberships/manage/cancel/link",
+      "value": "Mitgliedschaft und Abonnement kündigen"
+    },
+    {
+      "key": "option/PROLONG/additionalPeriods/endDate",
+      "value": "Neu gültig bis zum {formattedEndDate}"
+    },
+    {
+      "key": "option/PROLONG/additionalPeriods/REGULAR/title",
+      "value": "Verlängerung um ein Jahr"
+    },
+    {
+      "key": "option/PROLONG/additionalPeriods/BONUS/title",
+      "value": "Plus {days} zusätzliche Tage als Geschenk für Verlegerinnen des ersten Tages"
+    },
+    {
+      "key": "option/PROLONG/additionalPeriods/BONUS/explanation",
+      "value": "Damit werden im nächsten Jahr nicht mehr so viele Verlängerungen im Januar fällig. Wir verringern unser Klumpenrisiko."
```

Comment:
Shouldn't the bonus days be celebrated as a nice freebie for our most ardent fans – instead of as a sensible risk management strategy? I suggest we delete this section – it adds no useful information and looks like “fine print”.